views/test.ejs

```diff
@@ -36,6 +46,48 @@
 
         $scope.changelocale = function() {
           $scope.request.request.locale = $scope.locale;
+          $scope.request.context.System.apiEndpoint = $scope.apiEndpoint;
+
+          if ($scope.locale === 'en-US') {
+            $scope.request.context.System.apiEndpoint = 'https://api.amazonalexa.com/';
+          } else {
+            $scope.request.context.System.apiEndpoint = 'https://api.eu.amazonalexa.com';
```

Comment:
Nitpick, the one above is terminated with a slash, this one not. Should be the same.

More importantly, isn't that there're two languages supported and German is .eu? This would forward-default things to a specific region while I think their intent is the opposite. WDYT?